### DISCUSSION TITLE
ModuleNotFoundError: No module named 'mmcv.parallel'

### DISCUSSION THREAD

**Discussion by Aismart20170718** [open-mmlab/mmcv#2889] [Q&A]
mmcv==2.0.1 下    from mmcv.parallel import collate, scatter
报错ModuleNotFoundError: No module named 'mmcv.parallel'
请问怎么解决？

**Comment by Strickland302** [comment]
Have you solved the problem？
I'm facing the same one.
torch1.12.1
mmcv2.1.0
mmdet2.25.3

**Comment by Puiching-Memory** [comment]
此处的讨论可能会帮助到你：
https://github.com/open-mmlab/mmcv/issues/2761
